packages/graphback-keycloak-authz/README.md

```diff
@@ -0,0 +1,16 @@
+## Graphback Keycloak Authz
+<p align="center">
+  <img width="400" src="https://github.com/aerogear/graphback/raw/master/website/static/img/graphback.png">
+  <br/>
+  Auto generate database structure, <br/>
+  GraphQL Resolvers and Queries from GraphQL types 🚀
+</p>
+
+**Documentation**: https://graphback.dev
+**Repository**: https://github.com/aerogear/graphback/
+
+## Graphback Keycloak Authz
+
+Add Keycloak authorization to Graphback runtime applications
```

Comment:
Let's add some info how this can be consumed. 
Taking look and playing with datasync starter to see how we can do this.